default-templates/new-account/settings/serverSide.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix dct: <http://purl.org/dc/terms/>.
+@prefix pim: <http://www.w3.org/ns/pim/space#>.
+@prefix solid: <http://www.w3.org/ns/solid/terms#>.
+@prefix unit: <http://www.w3.invalid/ns#>.
+
+<>
+  a pim:ConfigurationFile;
+
+  dct:description "Administrative settings for the server that are only readable to the user." .
```

Comment:
Not the server, the pod?

Also I think with "only readable to the user" you mean "that the user can only read"?

Reply:
yeah